Collectives DO NOT MERGE :)

karissa opened the pull request.

karissa commented:
Zak it said dont merge on the header. :/ i wanted to squash a lot of the
commits.

On Friday, July 17, 2015, Zak Greene <notifications@github.com> wrote:

> Merged #55 <https://github.com/strikedebt/debtcollective-web/pull/55>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139>
> .
>


-- 
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>

karissa commented:
Its not a big deal though, but next time can we check in? and the person
who opened the pr should always be the person who merges it; just in case
there are significant breaking changes.

On Friday, July 17, 2015, Karissa McKelvey <karissa.mckelvey@gmail.com>
wrote:

> Zak it said dont merge on the header. :/ i wanted to squash a lot of the
> commits.
>
> On Friday, July 17, 2015, Zak Greene <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> Merged #55 <https://github.com/strikedebt/debtcollective-web/pull/55>.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139>
>> .
>>
>
>
> --
> Karissa McKelvey
> http://karissa.github.io/ <http://karissamck.com>
>
> Send me an encrypted message:
> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>
>

-- 
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>

zakgreene commented:
Ohh, sorry about that. I couldn’t figure out how that even happened at first, but it was because I pulled from heroku-staging, because it wouldn’t let me push to there if I didn’t. Sorry, I’m not exactly a pro at Git.

Z

On July 17, 2015 at 2:44:08 PM, Karissa McKelvey (notifications@github.com) wrote:

Its not a big deal though, but next time can we check in? and the person
who opened the pr should always be the person who merges it; just in case
there are significant breaking changes.

On Friday, July 17, 2015, Karissa McKelvey <karissa.mckelvey@gmail.com>
wrote:

> Zak it said dont merge on the header. :/ i wanted to squash a lot of the
> commits.
>
> On Friday, July 17, 2015, Zak Greene <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> Merged #55 <https://github.com/strikedebt/debtcollective-web/pull/55>.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139>
>> .
>>
>
>
> --
> Karissa McKelvey
> http://karissa.github.io/ <http://karissamck.com>
>
> Send me an encrypted message:
> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>
>

--
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
—
Reply to this email directly or view it on GitHub.

karissa commented:
It's ok!

For future reference, overriding on heroku-staging is ok, so you could do:

git push heroku-staging master -f

To push a different branch, do

git push heroku-staging mybranch:master -f

Rock on! I'm almost done with what I was working on, should be good by
Sunday.. we need to push the collectives/action page by tomorrow night but
now our changes are all mixed in together. Is the DTR broken, or is it
still usable and just needs tweaks? If its usable, I can just push whatever
you have with my changes to production tomorrow night and you can let me
know once you're ready to do another push.

On Fri, Jul 17, 2015 at 3:16 PM, Zak Greene <notifications@github.com>
wrote:

> Ohh, sorry about that. I couldn’t figure out how that even happened at
> first, but it was because I pulled from heroku-staging, because it wouldn’t
> let me push to there if I didn’t. Sorry, I’m not exactly a pro at Git.
>
> Z
>
> On July 17, 2015 at 2:44:08 PM, Karissa McKelvey (notifications@github.com)
> wrote:
>
> Its not a big deal though, but next time can we check in? and the person
> who opened the pr should always be the person who merges it; just in case
> there are significant breaking changes.
>
> On Friday, July 17, 2015, Karissa McKelvey <karissa.mckelvey@gmail.com>
> wrote:
>
> > Zak it said dont merge on the header. :/ i wanted to squash a lot of the
> > commits.
> >
> > On Friday, July 17, 2015, Zak Greene <notifications@github.com
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> >
> >> Merged #55 <https://github.com/strikedebt/debtcollective-web/pull/55>.
> >>
> >> —
> >> Reply to this email directly or view it on GitHub
> >> <
> https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139>
> >> .
> >>
> >
> >
> > --
> > Karissa McKelvey
> > http://karissa.github.io/ <http://karissamck.com>
> >
> > Send me an encrypted message:
> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> >
> >
>
> --
> Karissa McKelvey
> http://karissa.github.io/ <http://karissamck.com>
>
> Send me an encrypted message:
> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122379867>
> .
>



-- 
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>

karissa commented:
s/Sunday/saturday (tmrw)

On Fri, Jul 17, 2015 at 3:21 PM, Karissa McKelvey <
karissa.mckelvey@gmail.com> wrote:

> It's ok!
>
> For future reference, overriding on heroku-staging is ok, so you could do:
>
> git push heroku-staging master -f
>
> To push a different branch, do
>
> git push heroku-staging mybranch:master -f
>
> Rock on! I'm almost done with what I was working on, should be good by
> Sunday.. we need to push the collectives/action page by tomorrow night but
> now our changes are all mixed in together. Is the DTR broken, or is it
> still usable and just needs tweaks? If its usable, I can just push whatever
> you have with my changes to production tomorrow night and you can let me
> know once you're ready to do another push.
>
> On Fri, Jul 17, 2015 at 3:16 PM, Zak Greene <notifications@github.com>
> wrote:
>
>> Ohh, sorry about that. I couldn’t figure out how that even happened at
>> first, but it was because I pulled from heroku-staging, because it wouldn’t
>> let me push to there if I didn’t. Sorry, I’m not exactly a pro at Git.
>>
>> Z
>>
>> On July 17, 2015 at 2:44:08 PM, Karissa McKelvey (
>> notifications@github.com) wrote:
>>
>> Its not a big deal though, but next time can we check in? and the person
>> who opened the pr should always be the person who merges it; just in case
>> there are significant breaking changes.
>>
>> On Friday, July 17, 2015, Karissa McKelvey <karissa.mckelvey@gmail.com>
>> wrote:
>>
>> > Zak it said dont merge on the header. :/ i wanted to squash a lot of the
>> > commits.
>> >
>> > On Friday, July 17, 2015, Zak Greene <notifications@github.com
>> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>> >
>> >> Merged #55 <https://github.com/strikedebt/debtcollective-web/pull/55>.
>> >>
>> >> —
>> >> Reply to this email directly or view it on GitHub
>> >> <
>> https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139>
>> >> .
>> >>
>> >
>> >
>> > --
>> > Karissa McKelvey
>> > http://karissa.github.io/ <http://karissamck.com>
>> >
>> > Send me an encrypted message:
>> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> >
>> >
>>
>> --
>> Karissa McKelvey
>> http://karissa.github.io/ <http://karissamck.com>
>>
>> Send me an encrypted message:
>> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> —
>> Reply to this email directly or view it on GitHub.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122379867>
>> .
>>
>
>
>
> --
> Karissa McKelvey
> http://karissa.github.io/ <http://karissamck.com>
>
> Send me an encrypted message:
> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>



-- 
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>

zakgreene commented:
Sounds good. Looks like it mostly works except some of the checkboxes are off. I don’t see it linked from the site anymore, so if I can resolve just the bugs it seems like it’d be fine to push it to production. Will try and do that tonight or tomorrow.

Z


On July 17, 2015 at 3:22:41 PM, Karissa McKelvey (notifications@github.com) wrote:

s/Sunday/saturday (tmrw)

On Fri, Jul 17, 2015 at 3:21 PM, Karissa McKelvey <
karissa.mckelvey@gmail.com> wrote:

> It's ok!
>
> For future reference, overriding on heroku-staging is ok, so you could do:
>
> git push heroku-staging master -f
>
> To push a different branch, do
>
> git push heroku-staging mybranch:master -f
>
> Rock on! I'm almost done with what I was working on, should be good by
> Sunday.. we need to push the collectives/action page by tomorrow night but
> now our changes are all mixed in together. Is the DTR broken, or is it
> still usable and just needs tweaks? If its usable, I can just push whatever
> you have with my changes to production tomorrow night and you can let me
> know once you're ready to do another push.
>
> On Fri, Jul 17, 2015 at 3:16 PM, Zak Greene <notifications@github.com>
> wrote:
>
>> Ohh, sorry about that. I couldn’t figure out how that even happened at
>> first, but it was because I pulled from heroku-staging, because it wouldn’t
>> let me push to there if I didn’t. Sorry, I’m not exactly a pro at Git.
>>
>> Z
>>
>> On July 17, 2015 at 2:44:08 PM, Karissa McKelvey (
>> notifications@github.com) wrote:
>>
>> Its not a big deal though, but next time can we check in? and the person
>> who opened the pr should always be the person who merges it; just in case
>> there are significant breaking changes.
>>
>> On Friday, July 17, 2015, Karissa McKelvey <karissa.mckelvey@gmail.com>
>> wrote:
>>
>> > Zak it said dont merge on the header. :/ i wanted to squash a lot of the
>> > commits.
>> >
>> > On Friday, July 17, 2015, Zak Greene <notifications@github.com
>> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>> >
>> >> Merged #55 <https://github.com/strikedebt/debtcollective-web/pull/55>.
>> >>
>> >> —
>> >> Reply to this email directly or view it on GitHub
>> >> <
>> https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139>
>> >> .
>> >>
>> >
>> >
>> > --
>> > Karissa McKelvey
>> > http://karissa.github.io/ <http://karissamck.com>
>> >
>> > Send me an encrypted message:
>> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> >
>> >
>>
>> --
>> Karissa McKelvey
>> http://karissa.github.io/ <http://karissamck.com>
>>
>> Send me an encrypted message:
>> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> —
>> Reply to this email directly or view it on GitHub.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122379867>
>> .
>>
>
>
>
> --
> Karissa McKelvey
> http://karissa.github.io/ <http://karissamck.com>
>
> Send me an encrypted message:
> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>



--
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
—
Reply to this email directly or view it on GitHub.

zakgreene commented:
Question: if I’ve changed the name of some fields, but they’re consistent between the PDF and the <form>, do you need to change anything on your end or should they fill in?


On July 17, 2015 at 3:25:17 PM, Zak Greene (zak@zakgreene.com) wrote:

Sounds good. Looks like it mostly works except some of the checkboxes are off. I don’t see it linked from the site anymore, so if I can resolve just the bugs it seems like it’d be fine to push it to production. Will try and do that tonight or tomorrow.

Z


On July 17, 2015 at 3:22:41 PM, Karissa McKelvey (notifications@github.com) wrote:

s/Sunday/saturday (tmrw)

On Fri, Jul 17, 2015 at 3:21 PM, Karissa McKelvey <
karissa.mckelvey@gmail.com> wrote:

> It's ok!
>
> For future reference, overriding on heroku-staging is ok, so you could do:
>
> git push heroku-staging master -f
>
> To push a different branch, do
>
> git push heroku-staging mybranch:master -f
>
> Rock on! I'm almost done with what I was working on, should be good by
> Sunday.. we need to push the collectives/action page by tomorrow night but
> now our changes are all mixed in together. Is the DTR broken, or is it
> still usable and just needs tweaks? If its usable, I can just push whatever
> you have with my changes to production tomorrow night and you can let me
> know once you're ready to do another push.
>
> On Fri, Jul 17, 2015 at 3:16 PM, Zak Greene <notifications@github.com>
> wrote:
>
>> Ohh, sorry about that. I couldn’t figure out how that even happened at
>> first, but it was because I pulled from heroku-staging, because it wouldn’t
>> let me push to there if I didn’t. Sorry, I’m not exactly a pro at Git.
>>
>> Z
>>
>> On July 17, 2015 at 2:44:08 PM, Karissa McKelvey (
>> notifications@github.com) wrote:
>>
>> Its not a big deal though, but next time can we check in? and the person
>> who opened the pr should always be the person who merges it; just in case
>> there are significant breaking changes.
>>
>> On Friday, July 17, 2015, Karissa McKelvey <karissa.mckelvey@gmail.com>
>> wrote:
>>
>> > Zak it said dont merge on the header. :/ i wanted to squash a lot of the
>> > commits.
>> >
>> > On Friday, July 17, 2015, Zak Greene <notifications@github.com
>> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>> >
>> >> Merged #55 <https://github.com/strikedebt/debtcollective-web/pull/55>.
>> >>
>> >> —
>> >> Reply to this email directly or view it on GitHub
>> >> <
>> https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139>
>> >> .
>> >>
>> >
>> >
>> > --
>> > Karissa McKelvey
>> > http://karissa.github.io/ <http://karissamck.com>
>> >
>> > Send me an encrypted message:
>> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> >
>> >
>>
>> --
>> Karissa McKelvey
>> http://karissa.github.io/ <http://karissamck.com>
>>
>> Send me an encrypted message:
>> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> —
>> Reply to this email directly or view it on GitHub.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122379867>
>> .
>>
>
>
>
> --
> Karissa McKelvey
> http://karissa.github.io/ <http://karissamck.com>
>
> Send me an encrypted message:
> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>



--
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
—
Reply to this email directly or view it on GitHub.

karissa commented:
For the analytics piece (to export a csv), I have a file that defines the
headers. You'll need to add to the headers found here:

https://github.com/strikedebt/debtcollective-web/blob/master/be/proj/arcs/dtr_fields.json

It's not super important though as I'm sure the analytics piece will get
updated/changed later.



On Fri, Jul 17, 2015 at 3:26 PM, Zak Greene <notifications@github.com>
wrote:

> Question: if I’ve changed the name of some fields, but they’re consistent
> between the PDF and the <form>, do you need to change anything on your end
> or should they fill in?
>
>
>
> On July 17, 2015 at 3:25:17 PM, Zak Greene (zak@zakgreene.com) wrote:
>
> Sounds good. Looks like it mostly works except some of the checkboxes are
> off. I don’t see it linked from the site anymore, so if I can resolve just
> the bugs it seems like it’d be fine to push it to production. Will try and
> do that tonight or tomorrow.
>
> Z
>
>
> On July 17, 2015 at 3:22:41 PM, Karissa McKelvey (notifications@github.com)
> wrote:
>
> s/Sunday/saturday (tmrw)
>
> On Fri, Jul 17, 2015 at 3:21 PM, Karissa McKelvey <
> karissa.mckelvey@gmail.com> wrote:
>
> > It's ok!
> >
> > For future reference, overriding on heroku-staging is ok, so you could
> do:
> >
> > git push heroku-staging master -f
> >
> > To push a different branch, do
> >
> > git push heroku-staging mybranch:master -f
> >
> > Rock on! I'm almost done with what I was working on, should be good by
> > Sunday.. we need to push the collectives/action page by tomorrow night
> but
> > now our changes are all mixed in together. Is the DTR broken, or is it
> > still usable and just needs tweaks? If its usable, I can just push
> whatever
> > you have with my changes to production tomorrow night and you can let me
> > know once you're ready to do another push.
> >
> > On Fri, Jul 17, 2015 at 3:16 PM, Zak Greene <notifications@github.com>
> > wrote:
> >
> >> Ohh, sorry about that. I couldn’t figure out how that even happened at
> >> first, but it was because I pulled from heroku-staging, because it
> wouldn’t
> >> let me push to there if I didn’t. Sorry, I’m not exactly a pro at Git.
> >>
> >> Z
> >>
> >> On July 17, 2015 at 2:44:08 PM, Karissa McKelvey (
> >> notifications@github.com) wrote:
> >>
> >> Its not a big deal though, but next time can we check in? and the person
> >> who opened the pr should always be the person who merges it; just in
> case
> >> there are significant breaking changes.
> >>
> >> On Friday, July 17, 2015, Karissa McKelvey <karissa.mckelvey@gmail.com>
> >> wrote:
> >>
> >> > Zak it said dont merge on the header. :/ i wanted to squash a lot of
> the
> >> > commits.
> >> >
> >> > On Friday, July 17, 2015, Zak Greene <notifications@github.com
> >> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> >> >
> >> >> Merged #55 <https://github.com/strikedebt/debtcollective-web/pull/55
> >.
> >> >>
> >> >> —
> >> >> Reply to this email directly or view it on GitHub
> >> >> <
> >>
> https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139>
> >> >> .
> >> >>
> >> >
> >> >
> >> > --
> >> > Karissa McKelvey
> >> > http://karissa.github.io/ <http://karissamck.com>
> >> >
> >> > Send me an encrypted message:
> >> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> >> >
> >> >
> >>
> >> --
> >> Karissa McKelvey
> >> http://karissa.github.io/ <http://karissamck.com>
> >>
> >> Send me an encrypted message:
> >> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> >> —
> >> Reply to this email directly or view it on GitHub.
> >>
> >> —
> >> Reply to this email directly or view it on GitHub
> >> <
> https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122379867
> >
> >> .
> >>
> >
> >
> >
> > --
> > Karissa McKelvey
> > http://karissa.github.io/ <http://karissamck.com>
> >
> > Send me an encrypted message:
> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> >
>
>
>
> --
> Karissa McKelvey
> http://karissa.github.io/ <http://karissamck.com>
>
> Send me an encrypted message:
> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122384889>
> .
>



-- 
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>

zakgreene commented:
Ok, but it sounds like that won’t affect whether it can fill in the form. So there must be an inconsistency on my end. Thanks.


On July 17, 2015 at 3:32:19 PM, Karissa McKelvey (notifications@github.com) wrote:

For the analytics piece (to export a csv), I have a file that defines the
headers. You'll need to add to the headers found here:

https://github.com/strikedebt/debtcollective-web/blob/master/be/proj/arcs/dtr_fields.json

It's not super important though as I'm sure the analytics piece will get
updated/changed later.



On Fri, Jul 17, 2015 at 3:26 PM, Zak Greene <notifications@github.com>
wrote:

> Question: if I’ve changed the name of some fields, but they’re consistent
> between the PDF and the <form>, do you need to change anything on your end
> or should they fill in?
>
>
>
> On July 17, 2015 at 3:25:17 PM, Zak Greene (zak@zakgreene.com) wrote:
>
> Sounds good. Looks like it mostly works except some of the checkboxes are
> off. I don’t see it linked from the site anymore, so if I can resolve just
> the bugs it seems like it’d be fine to push it to production. Will try and
> do that tonight or tomorrow.
>
> Z
>
>
> On July 17, 2015 at 3:22:41 PM, Karissa McKelvey (notifications@github.com)
> wrote:
>
> s/Sunday/saturday (tmrw)
>
> On Fri, Jul 17, 2015 at 3:21 PM, Karissa McKelvey <
> karissa.mckelvey@gmail.com> wrote:
>
> > It's ok!
> >
> > For future reference, overriding on heroku-staging is ok, so you could
> do:
> >
> > git push heroku-staging master -f
> >
> > To push a different branch, do
> >
> > git push heroku-staging mybranch:master -f
> >
> > Rock on! I'm almost done with what I was working on, should be good by
> > Sunday.. we need to push the collectives/action page by tomorrow night
> but
> > now our changes are all mixed in together. Is the DTR broken, or is it
> > still usable and just needs tweaks? If its usable, I can just push
> whatever
> > you have with my changes to production tomorrow night and you can let me
> > know once you're ready to do another push.
> >
> > On Fri, Jul 17, 2015 at 3:16 PM, Zak Greene <notifications@github.com>
> > wrote:
> >
> >> Ohh, sorry about that. I couldn’t figure out how that even happened at
> >> first, but it was because I pulled from heroku-staging, because it
> wouldn’t
> >> let me push to there if I didn’t. Sorry, I’m not exactly a pro at Git.
> >>
> >> Z
> >>
> >> On July 17, 2015 at 2:44:08 PM, Karissa McKelvey (
> >> notifications@github.com) wrote:
> >>
> >> Its not a big deal though, but next time can we check in? and the person
> >> who opened the pr should always be the person who merges it; just in
> case
> >> there are significant breaking changes.
> >>
> >> On Friday, July 17, 2015, Karissa McKelvey <karissa.mckelvey@gmail.com>
> >> wrote:
> >>
> >> > Zak it said dont merge on the header. :/ i wanted to squash a lot of
> the
> >> > commits.
> >> >
> >> > On Friday, July 17, 2015, Zak Greene <notifications@github.com
> >> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> >> >
> >> >> Merged #55 <https://github.com/strikedebt/debtcollective-web/pull/55
> >.
> >> >>
> >> >> —
> >> >> Reply to this email directly or view it on GitHub
> >> >> <
> >>
> https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139>
> >> >> .
> >> >>
> >> >
> >> >
> >> > --
> >> > Karissa McKelvey
> >> > http://karissa.github.io/ <http://karissamck.com>
> >> >
> >> > Send me an encrypted message:
> >> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> >> >
> >> >
> >>
> >> --
> >> Karissa McKelvey
> >> http://karissa.github.io/ <http://karissamck.com>
> >>
> >> Send me an encrypted message:
> >> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> >> —
> >> Reply to this email directly or view it on GitHub.
> >>
> >> —
> >> Reply to this email directly or view it on GitHub
> >> <
> https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122379867
> >
> >> .
> >>
> >
> >
> >
> > --
> > Karissa McKelvey
> > http://karissa.github.io/ <http://karissamck.com>
> >
> > Send me an encrypted message:
> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> >
>
>
>
> --
> Karissa McKelvey
> http://karissa.github.io/ <http://karissamck.com>
>
> Send me an encrypted message:
> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122384889>
> .
>



--
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
—
Reply to this email directly or view it on GitHub.

karissa commented:
Ah, I forgot -- you had asked me to do some back-end filtering last time,
here:

https://github.com/strikedebt/debtcollective-web/blob/master/be/proj/arcs/corinthian.py#L185

On Fri, Jul 17, 2015 at 3:33 PM, Zak Greene <notifications@github.com>
wrote:

> Ok, but it sounds like that won’t affect whether it can fill in the form.
> So there must be an inconsistency on my end. Thanks.
>
>
>
> On July 17, 2015 at 3:32:19 PM, Karissa McKelvey (notifications@github.com)
> wrote:
>
> For the analytics piece (to export a csv), I have a file that defines the
> headers. You'll need to add to the headers found here:
>
>
> https://github.com/strikedebt/debtcollective-web/blob/master/be/proj/arcs/dtr_fields.json
>
> It's not super important though as I'm sure the analytics piece will get
> updated/changed later.
>
>
>
> On Fri, Jul 17, 2015 at 3:26 PM, Zak Greene <notifications@github.com>
> wrote:
>
> > Question: if I’ve changed the name of some fields, but they’re consistent
> > between the PDF and the <form>, do you need to change anything on your
> end
> > or should they fill in?
> >
> >
> >
> > On July 17, 2015 at 3:25:17 PM, Zak Greene (zak@zakgreene.com) wrote:
> >
> > Sounds good. Looks like it mostly works except some of the checkboxes are
> > off. I don’t see it linked from the site anymore, so if I can resolve
> just
> > the bugs it seems like it’d be fine to push it to production. Will try
> and
> > do that tonight or tomorrow.
> >
> > Z
> >
> >
> > On July 17, 2015 at 3:22:41 PM, Karissa McKelvey (
> notifications@github.com)
> > wrote:
> >
> > s/Sunday/saturday (tmrw)
> >
> > On Fri, Jul 17, 2015 at 3:21 PM, Karissa McKelvey <
> > karissa.mckelvey@gmail.com> wrote:
> >
> > > It's ok!
> > >
> > > For future reference, overriding on heroku-staging is ok, so you could
> > do:
> > >
> > > git push heroku-staging master -f
> > >
> > > To push a different branch, do
> > >
> > > git push heroku-staging mybranch:master -f
> > >
> > > Rock on! I'm almost done with what I was working on, should be good by
> > > Sunday.. we need to push the collectives/action page by tomorrow night
> > but
> > > now our changes are all mixed in together. Is the DTR broken, or is it
> > > still usable and just needs tweaks? If its usable, I can just push
> > whatever
> > > you have with my changes to production tomorrow night and you can let
> me
> > > know once you're ready to do another push.
> > >
> > > On Fri, Jul 17, 2015 at 3:16 PM, Zak Greene <notifications@github.com>
> > > wrote:
> > >
> > >> Ohh, sorry about that. I couldn’t figure out how that even happened at
> > >> first, but it was because I pulled from heroku-staging, because it
> > wouldn’t
> > >> let me push to there if I didn’t. Sorry, I’m not exactly a pro at Git.
> > >>
> > >> Z
> > >>
> > >> On July 17, 2015 at 2:44:08 PM, Karissa McKelvey (
> > >> notifications@github.com) wrote:
> > >>
> > >> Its not a big deal though, but next time can we check in? and the
> person
> > >> who opened the pr should always be the person who merges it; just in
> > case
> > >> there are significant breaking changes.
> > >>
> > >> On Friday, July 17, 2015, Karissa McKelvey <
> karissa.mckelvey@gmail.com>
> > >> wrote:
> > >>
> > >> > Zak it said dont merge on the header. :/ i wanted to squash a lot of
> > the
> > >> > commits.
> > >> >
> > >> > On Friday, July 17, 2015, Zak Greene <notifications@github.com
> > >> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> > >> >
> > >> >> Merged #55 <
> https://github.com/strikedebt/debtcollective-web/pull/55
> > >.
> > >> >>
> > >> >> —
> > >> >> Reply to this email directly or view it on GitHub
> > >> >> <
> > >>
> > https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139
> >
> > >> >> .
> > >> >>
> > >> >
> > >> >
> > >> > --
> > >> > Karissa McKelvey
> > >> > http://karissa.github.io/ <http://karissamck.com>
> > >> >
> > >> > Send me an encrypted message:
> > >> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> > >> >
> > >> >
> > >>
> > >> --
> > >> Karissa McKelvey
> > >> http://karissa.github.io/ <http://karissamck.com>
> > >>
> > >> Send me an encrypted message:
> > >> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> > >> —
> > >> Reply to this email directly or view it on GitHub.
> > >>
> > >> —
> > >> Reply to this email directly or view it on GitHub
> > >> <
> >
> https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122379867
> > >
> > >> .
> > >>
> > >
> > >
> > >
> > > --
> > > Karissa McKelvey
> > > http://karissa.github.io/ <http://karissamck.com>
> > >
> > > Send me an encrypted message:
> > > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> > >
> >
> >
> >
> > --
> > Karissa McKelvey
> > http://karissa.github.io/ <http://karissamck.com>
> >
> > Send me an encrypted message:
> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> > —
> > Reply to this email directly or view it on GitHub.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122384889
> >
> > .
> >
>
>
>
> --
> Karissa McKelvey
> http://karissa.github.io/ <http://karissamck.com>
>
> Send me an encrypted message:
> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122386734>
> .
>



-- 
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>

karissa commented:
I just created new migrations, so if you're getting database errors you
need to run

python manage.py migrate collectives

or on heroku-staging:

heroku run --app=debt-is-staging python be/manage.py migrate collectives

On Fri, Jul 17, 2015 at 3:34 PM, Karissa McKelvey <
karissa.mckelvey@gmail.com> wrote:

> Ah, I forgot -- you had asked me to do some back-end filtering last time,
> here:
>
>
> https://github.com/strikedebt/debtcollective-web/blob/master/be/proj/arcs/corinthian.py#L185
>
> On Fri, Jul 17, 2015 at 3:33 PM, Zak Greene <notifications@github.com>
> wrote:
>
>> Ok, but it sounds like that won’t affect whether it can fill in the form.
>> So there must be an inconsistency on my end. Thanks.
>>
>>
>>
>> On July 17, 2015 at 3:32:19 PM, Karissa McKelvey (
>> notifications@github.com) wrote:
>>
>> For the analytics piece (to export a csv), I have a file that defines the
>> headers. You'll need to add to the headers found here:
>>
>>
>> https://github.com/strikedebt/debtcollective-web/blob/master/be/proj/arcs/dtr_fields.json
>>
>> It's not super important though as I'm sure the analytics piece will get
>> updated/changed later.
>>
>>
>>
>> On Fri, Jul 17, 2015 at 3:26 PM, Zak Greene <notifications@github.com>
>> wrote:
>>
>> > Question: if I’ve changed the name of some fields, but they’re
>> consistent
>> > between the PDF and the <form>, do you need to change anything on your
>> end
>> > or should they fill in?
>> >
>> >
>> >
>> > On July 17, 2015 at 3:25:17 PM, Zak Greene (zak@zakgreene.com) wrote:
>> >
>> > Sounds good. Looks like it mostly works except some of the checkboxes
>> are
>> > off. I don’t see it linked from the site anymore, so if I can resolve
>> just
>> > the bugs it seems like it’d be fine to push it to production. Will try
>> and
>> > do that tonight or tomorrow.
>> >
>> > Z
>> >
>> >
>> > On July 17, 2015 at 3:22:41 PM, Karissa McKelvey (
>> notifications@github.com)
>> > wrote:
>> >
>> > s/Sunday/saturday (tmrw)
>> >
>> > On Fri, Jul 17, 2015 at 3:21 PM, Karissa McKelvey <
>> > karissa.mckelvey@gmail.com> wrote:
>> >
>> > > It's ok!
>> > >
>> > > For future reference, overriding on heroku-staging is ok, so you could
>> > do:
>> > >
>> > > git push heroku-staging master -f
>> > >
>> > > To push a different branch, do
>> > >
>> > > git push heroku-staging mybranch:master -f
>> > >
>> > > Rock on! I'm almost done with what I was working on, should be good by
>> > > Sunday.. we need to push the collectives/action page by tomorrow night
>> > but
>> > > now our changes are all mixed in together. Is the DTR broken, or is it
>> > > still usable and just needs tweaks? If its usable, I can just push
>> > whatever
>> > > you have with my changes to production tomorrow night and you can let
>> me
>> > > know once you're ready to do another push.
>> > >
>> > > On Fri, Jul 17, 2015 at 3:16 PM, Zak Greene <notifications@github.com
>> >
>> > > wrote:
>> > >
>> > >> Ohh, sorry about that. I couldn’t figure out how that even happened
>> at
>> > >> first, but it was because I pulled from heroku-staging, because it
>> > wouldn’t
>> > >> let me push to there if I didn’t. Sorry, I’m not exactly a pro at
>> Git.
>> > >>
>> > >> Z
>> > >>
>> > >> On July 17, 2015 at 2:44:08 PM, Karissa McKelvey (
>> > >> notifications@github.com) wrote:
>> > >>
>> > >> Its not a big deal though, but next time can we check in? and the
>> person
>> > >> who opened the pr should always be the person who merges it; just in
>> > case
>> > >> there are significant breaking changes.
>> > >>
>> > >> On Friday, July 17, 2015, Karissa McKelvey <
>> karissa.mckelvey@gmail.com>
>> > >> wrote:
>> > >>
>> > >> > Zak it said dont merge on the header. :/ i wanted to squash a lot
>> of
>> > the
>> > >> > commits.
>> > >> >
>> > >> > On Friday, July 17, 2015, Zak Greene <notifications@github.com
>> > >> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>> > >> >
>> > >> >> Merged #55 <
>> https://github.com/strikedebt/debtcollective-web/pull/55
>> > >.
>> > >> >>
>> > >> >> —
>> > >> >> Reply to this email directly or view it on GitHub
>> > >> >> <
>> > >>
>> >
>> https://github.com/strikedebt/debtcollective-web/pull/55#event-358748139>
>> > >> >> .
>> > >> >>
>> > >> >
>> > >> >
>> > >> > --
>> > >> > Karissa McKelvey
>> > >> > http://karissa.github.io/ <http://karissamck.com>
>> > >> >
>> > >> > Send me an encrypted message:
>> > >> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> > >> >
>> > >> >
>> > >>
>> > >> --
>> > >> Karissa McKelvey
>> > >> http://karissa.github.io/ <http://karissamck.com>
>> > >>
>> > >> Send me an encrypted message:
>> > >> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> > >> —
>> > >> Reply to this email directly or view it on GitHub.
>> > >>
>> > >> —
>> > >> Reply to this email directly or view it on GitHub
>> > >> <
>> >
>> https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122379867
>> > >
>> > >> .
>> > >>
>> > >
>> > >
>> > >
>> > > --
>> > > Karissa McKelvey
>> > > http://karissa.github.io/ <http://karissamck.com>
>> > >
>> > > Send me an encrypted message:
>> > > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> > >
>> >
>> >
>> >
>> > --
>> > Karissa McKelvey
>> > http://karissa.github.io/ <http://karissamck.com>
>> >
>> > Send me an encrypted message:
>> > https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> > —
>> > Reply to this email directly or view it on GitHub.
>> >
>> > —
>> > Reply to this email directly or view it on GitHub
>> > <
>> https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122384889
>> >
>> > .
>> >
>>
>>
>>
>> --
>> Karissa McKelvey
>> http://karissa.github.io/ <http://karissamck.com>
>>
>> Send me an encrypted message:
>> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>> —
>> Reply to this email directly or view it on GitHub.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/strikedebt/debtcollective-web/pull/55#issuecomment-122386734>
>> .
>>
>
>
>
> --
> Karissa McKelvey
> http://karissa.github.io/ <http://karissamck.com>
>
> Send me an encrypted message:
> https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
>



-- 
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
